desktop/analytics/artwork.js

```diff
@@ -39,6 +39,15 @@
         context_type: context.context_type
       })
     })
+    .on('click', '.artwork-auction__live-button a', function () {
+      analytics.track('click', {
+        type: 'button',
+        label: $(this).text(),
+        flow: 'artworks',
+        context_module: 'artwork metadata',
+        destination_path: $(this).attr('href') // perhaps import sd so we can remove the domain?
```

Comment:
safe to use `sd` here?

Reply:
Yep yep. We pass references to `sd` in various analytics calls.